pkg/openshift/certgen/unstable/defaults.go

```diff
@@ -17,15 +17,25 @@ func OpenShiftSetDefaultCerts(a *api.Properties, orchestratorName, clusterID str
 		return true, nil
 	}
 
+	var ips []net.IP
+	var dnsPrefix string
+	if a.MasterProfile != nil {
+		ips = []net.IP{net.ParseIP(a.MasterProfile.FirstConsecutiveStaticIP)}
+		dnsPrefix = a.MasterProfile.DNSPrefix
+	} else if a.HostedMasterProfile == nil {
+		return false, fmt.Errorf("no masterProfile or hostedMasterProfile found")
```

Comment:
Right, now this code needs to get called for generating the node boostrap certificate (and I think I need to make a change and expect the master CA as an input instead of generating it here) but all of this changeset depends on what happens in the RP, ie. how are we going to generate certs for HCP.

cc @sozercan @khenidak